elliott/elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -261,16 +270,14 @@ def get_assembly_bug_ids(runtime, bug_tracker_type):
 
 
 def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int],
-                            permitted_bug_ids, permissive=False, major_version: int = 4):
+                            permitted_bug_ids, operator_advisory, permissive=False, major_version: int = 4):
     bugs_by_type: Dict[str, type_bug_set] = {
         "rpm": set(),
         "image": set(),
         "extras": set(),
-        # Metadata advisory will not have Bugs for z-stream releases
-        # But at GA time it can have operator builds for the early operator release
-        # and thus related extras bugs (including trackers and flaws) will need to be attached to it
-        # see: https://art-docs.engineering.redhat.com/release/4.y-ga/#early-silent-operator-release
-        "metadata": set(),
+        # If operator_advisory is set to 'advance' we consider the advance advisory as the metadata advisory
+        # advance advisory will have bugs while metadata advisory will not, until GA.
+        f"{operator_advisory}": set(),
```

Comment:
don't need an f-string quote here since operator_advisory is a string

Reply:
I'm thinking we don't explicitly need to know the concept of an advance release here.
If you see the code, we're fetching advisories from `advisory_id_map` and then wherever the build is attached we assign the bug there. So we can do something like

```python
bugs_by_type: Dict[str, type_bug_set] = {kind: set() for kind in advisory_id_map.keys()}
```

Reply:
added in https://github.com/openshift-eng/art-tools/commit/65be8eaf67986030464a864aa735a1cd755bfdc9